common/app/views/support/GetClasses.scala

```diff
@@ -25,7 +26,7 @@ object GetClasses {
       ("fc-item--has-boosted-title", item.displaySettings.showBoostedHeadline),
       ("fc-item--live", item.isLive),
       ("fc-item--has-metadata",
-        item.timeStampDisplay.isDefined || item.discussionSettings.isCommentable),
+        item.timeStampDisplay.isDefined || item.discussionSettings.isCommentable || SaveForLaterSwitch.isSwitchedOn),
```

Comment:
Cos the delete button goes in the item meta and if there ain't a timestamp or a discussion it dosnae wark